blockstack/lib/nameset/db.py

```diff
@@ -1012,6 +1012,40 @@ def namedb_patch_genesis(con, merge_block_height, genesis_block_add, genesis_blo
 
     namedb_query_execute(con, "END", ())
 
+def namedb_patch_v2_upgrade_signal(con):
+    """
+    Simple db "migration", add the v2 upgrade metadata table
```

Comment:
I couldn't find any existing sqlite migration code, so added a quick patch here. Let me know if there's a better way to go about this.

Reply:
This is fine -- the table will end up in the `blockstack-server.db` file, right?